windows/tunnel/service.go

```diff
@@ -109,17 +109,13 @@ func (service *tunnelService) Execute(args []string, r <-chan svc.ChangeRequest,
 		return
 	}
 
-	config, err = conf.LoadFromPath(service.Path)
+	config, err = conf.FromWgQuickWithUnknownEncoding(service.ConfString,"MozillaVPN")
```

Comment:
It feels incorrect to hard-code the tunnel interface name here, this is something that should be passed in by the daemon as a runtime parameter.